addon/-private/normalize-locale.ts

```diff
@@ -0,0 +1,15 @@
+import { assert } from '@ember/debug';
+
+function assertIsString(val: any): asserts val is string {
```

Comment:
`asserts val is string` - Is this a typescript feature?  If so, pretty awesome!

Reply:
a brand new feature! https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#assertion-functions